### ISSUE TITLE
Remember Input Mode

### ISSUE THREAD

**Issue by GoogleCodeExporter** [thaindq/opensudoku-android#16]
```
What steps will reproduce the problem?
1. Play a puzzle with your favorite input mode
2. Come back later
3. Start playing again

It should remember your last-used input method, or have a preference for
which to start with.  Instead, it always resets to the pop up method.

Using 0.6.1, android 1.5


```

Original issue reported on code.google.com by `jhkni...@gmail.com` on 10 Aug 2009 at 5:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 31 Aug 2009 at 3:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 7 Sep 2009 at 5:43
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Solved in 0.7.0. You can wait for this release or download test version here:
http://code.google.com/p/opensudoku-android/downloads/list (look for OpenSudoku 
0.7.0
- unstable).
```

Original comment by `romario...@gmail.com` on 21 Sep 2009 at 5:25
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 1 Oct 2009 at 9:06
- Changed state: **Verified**
